src/cdomains/intDomain.ml

```diff
@@ -349,6 +349,446 @@ struct
     | None -> None
 end
 
+module IntPair =
```

Comment:
Is this necessary? Would it not be better to use Interval32 here?

Reply:
I am not sure if I want to have one one Option type inside of another Option type — a set of intervals contaioning None values doesn't look lik a right thing

Reply:
True, but maybe you can somehow pull out the common thing into some sort of module that is then included in the module you have in your set and in `Interval32`? 
It seems like you are duplicating a lot of code with having your intervals being completely separate form the ones in `Interval32`.

Reply:
Ah, that looks like a right thing to do.